Containerfile

```diff
@@ -1,32 +1,20 @@
-FROM scratch AS files
```

Comment:
Could have been a separate commit/PR

Reply:
I will plan to move it to its own commit. I made this change to make debugging faster as I didn't have to reinstall all dependencies every time I modified the scripts.